/joy_remapper_node:
  ros__parameters:
    from:
      # Buttons
      A: 0
      B: 1
      X: 2
      Y: 3
      LB: 4
      RB: 5
      LT_BTN: 6
      RT_BTN: 7
      VIEW: 8
      MENU: 9
      LS: 10
      RS: 11
      D_PAD_UP: 12
      D_PAD_DOWN: 13
      D_PAD_LEFT: 14
      D_PAD_RIGHT: 15
      XBOX: 16
      SHARE: 17
      # Axes
      LS_X: 0
      LS_Y: 1
      RS_X: 2
      RS_Y: 3
    to:
      # Buttons
      A: 0
      B: 1
      X: 2
      Y: 3
      LB: 4
      RB: 5
      VIEW: 6
      MENU: 7
      XBOX: 8
      LS: 9
      RS: 10
      SHARE: 11
      # Axes
      LS_X: 0
      LS_Y: 1
      LT: 2
      RS_X: 3
      RS_Y: 4
      RT: 5
      D_PAD_X: 6
      D_PAD_Y: 7
